Packs/Core/ReleaseNotes/2_0_13.md

```diff
@@ -0,0 +1,8 @@
+
+#### Playbooks
+
+##### NGFW Scan
+
+- Added the *'UserVerification'* playbook input to determine whether user verification is required before blocking malicious IP addresses using the **'Block IP - Generic v3'** sub-playbook.
+
+- Updated the configuration for the *'UserVerification'* input within the **'Block IP - Generic v3'** sub-playbook.
```

Comment:
From false to true? What do you mean with updated the configuration?

Reply:
The configs for the UserVerification input within Block IP - Generic v3 sub-playbook was changed. Before it was set to 'true' and now it is set to the value of UserVerification input within NGFW Scan playbook

Reply:
fixed